html/docs/start.html

```diff
@@ -394,7 +394,7 @@
             <br />
             <a href="mailto:support@taskwarrior.org"><span class="glyphicon glyphicon-envelope"></span> Email</a>
             <br />
-            <a href="https://twitter.com/taskwarrior"><img src="/images/twitter_dark.png" width="16px" height="16px"> Twitter</a>
+            <a href="https://twitter.com/taskwarrior"><img src="/images/twitter_dark.png" alt="" width="16px" height="16px"> Twitter</a>
```

Comment:
Out of curiosity, what is the motivation behind having empty alt?

Reply:
Sonar Lint complaint: "Image, area and button with image tags should have an "alt" attribute"

> The alt attribute provides a textual alternative to an image. ...
> It is also very important to not set an alt attribute to a non-informative value. For example <img ... alt="logo"> is useless as it doesn't give any information to the user. In this case, as for any other decorative image, it is better to use a CSS background image instead of an <img> tag. If using CSS background-image is not possible, an empty alt="" is tolerated.